refresh/index.js

```diff
@@ -666,10 +666,17 @@ module.exports = Generator.extend({
   parseFromSwagger: function () {
     if (!this.fromSwagger) return
 
-    return swaggerize.parse(this.fs, this.fromSwagger)
-      .then(response => {
-        this.loadedApi = response.loaded
-        this.parsedSwagger = response.parsed
+    return helpers.loadAsync(this.fromSwagger, this.fs)
+      .then(loaded => {
+        return swaggerize.parse(loaded)
```

Comment:
This could be shortened to `.then(swaggerize.parse)` or keeping the explicit argument `.then(loaded => swaggerize.parse(loaded))`